heft-plugins/heft-webpack4-plugin/package.json

```diff
@@ -9,7 +9,7 @@
   },
   "homepage": "https://rushstack.io/pages/heft/overview/",
   "main": "lib/index.js",
-  "types": "dist/heft-webpack4-plugin.d.ts",
+  "types": "lib/index.d.ts",
```

Comment:
Why don't we want to point at the API Extractor rollup?

Reply:
This is because the types generated by API Extractor end up failing to load correctly due to some issue in the WebpackDevServer typings. I forget the exact reasoning, though @dmichon-msft can expand on this.

Reply:
@thelarkinn - Can you take a look at this issue?